CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ All notable changes to Kafka extension will be documented in this file.
 - Internal topics are now hidden by default. See [#29](https://github.com/jlandersen/vscode-kafka/issues/29) and [#74](https://github.com/jlandersen/vscode-kafka/pull/74).
 - Elements are now sorted alphabetically in the Kafka explorer. See [#63](https://github.com/jlandersen/vscode-kafka/issues/63).
 - Clusters are now sorted in the cluster selection wizard. See [#83](https://github.com/jlandersen/vscode-kafka/issues/83).
+- Selection of cluster is now visible in the explorer and `Select Cluster` menu item is displayed only for unselected cluster. See [#82](https://github.com/jlandersen/vscode-kafka/issues/82).
```

Comment:
Cluster selection is now ... the `Select Cluster` menu is displayed only for unselected clusters.

Reply:
fixed